### PR TITLE
Add componentWillReceiveProps to ResizableBox

### PR DESCRIPTION
**Pull request by JoaoMosmann**
ResizableBox should be able to react to prop changes as Resizable component do.

### CODE REVIEW

**Comment by JoaoMosmann** [comment]
Any idea when this PR can be evaluated ?

File: lib/Resizable.jsx [hunk 1]
```diff
@@ -146,6 +146,7 @@ export default class Resizable extends React.Component {
       const newState = {};
       if (handlerName === 'onResizeStart') {
         newState.resizing = true;
+        e.persist && e.persist()
```
Comment (STRML) [inline lib/Resizable.jsx]: Why was this change made?

**Comment by STRML** [comment]
I'm fine with this provided you address the comment re: event persistence.

**Comment by mmmeff** [comment]
@JoaoMosmann ❓

**Comment by JoaoMosmann** [comment]
Sure. done

**Comment by STRML** [comment]
@JoaoMosmann That's not what I was asking; I was asking "why are you persisting the event?", not "what does e.persist() mean?"

**Comment by rahilsondhi** [comment]
@JoaoMosmann any update on @STRML's question? He's just asking **why** `e.persist && e.persist()` is needed.

I would love to see this get merged.

**Comment by JoaoMosmann** [comment]
It's explained in #45

**Pull request closed without merging**